.env.example

```diff
@@ -1,14 +1,26 @@
 # BOT
 SMF_BOT_BACKEND_URL="http://localhost:5555"
 SMF_BOT_DRIP_AMOUNT=10
+
+# Log in to the bots matrix account. Click on the name in the top left corner, then "Settings".
+# Click the "Help & About" tab (left side of the dialog).
+# Scroll to the bottom and click on <click to reveal> part of Access Token.
+# Once you logout - access token invalidates, so keep somewhere bot account logged in
 SMF_BOT_MATRIX_ACCESS_TOKEN="NotValidExampleToken"
-SMF_BOT_MATRIX_BOT_USER_ID="@NotValidExampleAccount:matrix.org"
+
+# Make sure to create an account within matrix.parity.io. You can use yourname+matrixbot@parity.io
```

Comment:
Is this `matrixbot@parity.io` a real email or just an example? It's confusing

It could be explained like this

```suggestion
# See https://github.com/paritytech/devops/wiki/Matrix%3A-Registering-a-Matrix-Bot for how to register a bot
# Note: The bot account needs to be created in SMF_BOT_MATRIX_SERVER
```